libsplinter/src/circuit/routing/mod.rs

```diff
@@ -180,7 +180,7 @@ impl fmt::Display for ServiceId {
 
 impl Eq for ServiceId {}
 
-/// The trait that defines a writer for updating the in-memory routing table
+/// The trait that defines a writer for updating the routing table
```

Comment:
Shouldn't start with "The trait....", which is redundant info. Maybe just "Interface for updating the routing table" or something along those lines, similar for reader.